app/photosketchingcontroller.cpp

```diff
@@ -123,7 +123,7 @@ void PhotoSketchingController::clear()
 void PhotoSketchingController::backupSketches()
 {
   const QString photoFileName = QUrl( mPhotoSource ).fileName();
-  const QString photoPath = QString( "%1/%2/%3" ).arg( QDir::temp().absolutePath(), mProjectName, photoFileName );
```

Comment:
Shouldn't we update https://github.com/MerginMaps/mobile/pull/4081/files#diff-dbeaddca911ca07c0885e8dc5af9844d0c3dfce4395d5939a9d6dcaefa59677cR253 in the same manner (line 253)?